frontend/device/android/powerd.lua

```diff
@@ -69,6 +74,12 @@ function AndroidPowerD:turnOnFrontlightHW(done_callback)
 
     android.setScreenBrightness(math.floor(self.fl_intensity * self.bright_diff / self.fl_max))
 
+    -- Devices using OnyxWarmthController also need to turn on warmth
```

Comment:
Please remove the comment. It is not accurate. Code is enough to understand what happens and is 100% true.